acceptance_tests/features/environment.py

```diff
@@ -47,38 +37,54 @@ def after_step(context, step):
         logger.exception('Failed step', scenario=context.scenario.name, step=step.name, html=browser.html)
 
 
+def after_all(context):
+    browser.quit()
+
+
 def execute_collection_exercises():
-    test_file = 'resources/sample_files/business-survey-sample-date.csv'
-    logger.info('Loading sample', survey='bricks', period='201801')
-    sample_controller.load_sample('bricks', '201801', test_file)
-    logger.info('Loading sample', survey='bricks', period='201812')
-    sample_controller.load_sample('bricks', '201812', test_file)
-    logger.info('Loading sample', survey='QBS', period='1809')
-    sample_controller.load_sample('QBS', '1809', test_file)
-    logger.info('Executing collection exercise', survey_id='cb8accda-6118-4d3b-85a3-149e28960c54', period='201801')
-    collection_exercise_controller.execute_collection_exercise(survey_id='cb8accda-6118-4d3b-85a3-149e28960c54',
-                                                               period='201801')
-    logger.info('Executing collection exercise', survey_id='cb8accda-6118-4d3b-85a3-149e28960c54', period='201812')
-    collection_exercise_controller.execute_collection_exercise(survey_id='cb8accda-6118-4d3b-85a3-149e28960c54',
-                                                               period='201812')
-    logger.info('Executing collection exercise', survey_id='02b9c366-7397-42f7-942a-76dc5876d86d', period='1809')
-    collection_exercise_controller.execute_collection_exercise(survey_id='02b9c366-7397-42f7-942a-76dc5876d86d',
-                                                               period='1809')
-    logger.info('Waiting for collection exercises execution process to finish',
-                survey_id='cb8accda-6118-4d3b-85a3-149e28960c54', period='201801')
+    logger.info('Executing collection exercises')
+    # Bricks
+    execute_collection_exercise('cb8accda-6118-4d3b-85a3-149e28960c54', '201801')
+    execute_collection_exercise('cb8accda-6118-4d3b-85a3-149e28960c54', '201812')
+    # QBS
+    execute_collection_exercise('02b9c366-7397-42f7-942a-76dc5876d86d', '1809', ci_type='eQ')
+
+    logger.info('Waiting for collection exercises to finish executing')
     poll_database_for_iac(survey_id='cb8accda-6118-4d3b-85a3-149e28960c54', period='201801')
-    logger.info('Waiting for collection exercises execution process to finish',
-                survey_id='cb8accda-6118-4d3b-85a3-149e28960c54', period='201812')
     poll_database_for_iac(survey_id='cb8accda-6118-4d3b-85a3-149e28960c54', period='201812')
-    logger.info('Waiting for collection exercises execution process to finish',
-                survey_id='02b9c366-7397-42f7-942a-76dc5876d86d', period='1809')
     poll_database_for_iac(survey_id='02b9c366-7397-42f7-942a-76dc5876d86d', period='1809')
+    logger.info('Collection exercises have finished executing')
+
+
+def execute_collection_exercise(survey_id, period, ci_type='SEFT'):
```

Comment:
Is this setup pretty stable now? Used to fail sometime hence the `logger.exception('Failed to setup before running tests', html=browser.html)`

Reply:
This should be more stable now that we're not using the UI the execute the CE's